### ISSUE TITLE
Additional Lap information causes Server Error 500

### ISSUE THREAD

**Issue by campino2k** [Runalyze/Runalyze#1294]
This occures under following circumstance: I record a training (bike) session via my Forerunner 15. Changing activity to Biking, there seems the problem that biking is measured in km/h instead of min/km.

Changing activity type back to running, everything is fine, rounds are shown as expected.

```
Fatal error: Uncaught exception 'InvalidArgumentException' with message 'Pace objects must have the same unit.' in /[path]/inc/core/Activity/Pace.php:275

Stack trace: 
#0 /[path]/inc/core/View/Window/Laps/Table.php(150): Runalyze\Activity\Pace->compareTo(Object(Runalyze\Activity\Pace)) 
#1 /[path]/inc/core/View/Window/Laps/Table.php(127): Runalyze\View\Window\Laps\Table->tableRowFor(0) 
#2 /[path]/inc/core/View/Window/Laps/Table.php(92): Runalyze\View\Window\Laps\Table->tableBody() 
#3 /[path]/inc/core/View/Window/Laps/Window.php(242): Runalyze\View\Window\Laps\Table->display() 
#4 /[path]/inc/core/View/Window/Laps/Window.php(202): Runalyze\View\Window\Laps\Window->displayTable() 
#5 /[path]/call/call.Training.roundsInfo.php(14): Runalyze\View\Window\Laps\Window->display() 
#6 {main} thrown in /[path]/inc/core/Activity/Pace.php on line 275
```

**Comment by campino2k** [comment]
Addition: Changing biking to mesurement in min/km, the rounds are also shown.

I would expect to see the same stats per km when viewing biking.

**Comment by campino2k** [comment]
This seems to be an duplicate of #1271 and #1273 and is fixed with 2.0.1
